openshift/templates/api/api-deploy.overrides.sh

```diff
@@ -20,6 +20,7 @@ if createOperation; then
   readParameter "OIDC_RP_CLIENT_SECRET - Please provide the OIDC RP Client Secret.  The default is a blank string." OIDC_RP_CLIENT_SECRET "" "false"
 
   # Get the eFiling settings
+  readParameter "EFILING_ENABLED - Please provide a boolean to enable eFiling.  The default is false." EFILING_ENABLED "" "false"
```

Comment:
Since this setting does not contain any sensitive information there is no need to collect the setting on the command line and store it as a secret.  So you can remove the updates from the override scripts.